Tasks/AzureRmWebAppDeployment/task.json

```diff
@@ -375,6 +372,8 @@
         "ScriptStatusTimeout": "Unable to fetch script status due to timeout.",
         "PollingForFileTimeOut": "Unable to fetch script status due to timeout. You can increase the timeout limit by setting 'appservicedeploy.retrytimeout' variable to number of minutes required.",
         "InvalidPollOption": "Invalid polling option provided: %s.",
-        "MissingWebConfigParameters": "Some web.config parameters are missing"
+        "MissingWebConfigParameters": "Some web.config parameters are missing",
+        "appServiceWarmupedUpSuccessfully": "Successfully warmed up azure app service.",
```

Comment:
I think the warm messages are only debug messages. The are not user facing and need not be localized.